dspace-api/src/main/java/org/dspace/checker/MostRecentChecksum.java

```diff
@@ -61,6 +64,35 @@
     @JoinColumn(name= "result")
     private ChecksumResult checksumResult;
 
+    public MostRecentChecksum(Bitstream bitstream) {
```

Comment:
I would move this to the MostRecentChecksumService, we want to keep our Data objects as clean as possible.